### PR TITLE
New Fonts Added

### PR DESCRIPTION
**Pull request by katieleebrown**
Fonts pulled from potential_fonts.txt and cross-referenced with existing database. Fonts with multiple characters have been submitted in array format. Names have not been added for fonts, but can be updated in a follow up request if you'd like. This is in reference to Add More Fonts issue #3 : https://github.com/waterrmalann/fancy-fonts-generator/issues/3.

### CODE REVIEW

File: fonts.json [hunk 1]
```diff
@@ -172,5 +172,125 @@
...
+        "fontName": "",
+        "fontLower": ["【a】","【b】", "【c】", "【d】","【e】","【f】","【g】","【h】","【i】","【j】","【k】","【l】","【m】","【n】","【o】","【p】","【q】","【r】","【s】","【t】","【u】","【v】","【w】","【x】","【y】","【z】"],
+        "fontUpper": ["【A】","【B】","【C】","【D】","【E】","【F】","【G】","【H】","【I】","【J】","【K】","【L】","【M】","【N】","【O】","【P】","【Q】","【R】","【S】","【T】","【U】","【V】","【W】","【X】","【Y】","【Z】"],
+        "fontDigits": ["【0】","【1】","【2】","【3】","【4】","【5】","【6】","【7】","【8】","【9】"]
+    },
+    {
+        "fontName": "",
+        "fontLower": ["『a』","『b』","『c』","『d』","『e』","『f』","『g』","『h』","『i』","『j』","『k』","『l』","『m』","『n』","『o』","『p』","『q』","『r』","『s』","『t』","『u』","『v』","『w』","『x』","『y』","『z』"],
+        "fontUpper": ["『A』","『B』","『C』","『D』","『E』","『F』","『G』","『H』","『I』","『J』","『K』","『L』","『M』","『N』","『O』","『P』","『Q』","『R』","『S』","『T』","『U』","『V』","『W』","『X』","『Y』","『Z』"],
+        "fontDigits": ["『0』'<'『1』","『2』","『3』","『4』","『5』","『6』","『7』","『8』","『9』"]
```
Comment (waterrmalann) [inline fonts.json:234]: Incorrect formatting, the numbers won't show up properly.

Comment (waterrmalann) [reply fonts.json:234]: image

**Review by waterrmalann — CHANGES_REQUESTED** [review]
Most of the the fonts work perfectly. Great work!

*However...*

- There is a minor formatting error on line 234 that prevents digits from being parsed properly.

- Also the fonts defined from line 248 onwards won't work. Though not immediately obvious, they are actually multi-character letters, you could check this for yourself by copying one letter into a JavaScript or Python interpreter as a string and checking it's length. They would also need to use the array-based syntax. I forgot to mention this potential headache in the issue. So because of that, the fonts give messed up outputs.

image
*When given input "The quick brown fox jumps over the lazy dog."*

If you could fix these two, I'll happily merge the pull request 😃

If the second issue is not easily fixable and you are unable to properly select the the letters individually from the string to split into arrays, then leave it as is and I'll take care of it. I may actually need to update the pseudofont parser to specifically support these fonts as they're a bit more weirder than the previous ones.

**Comment by waterrmalann** [comment]
> Names have not been added for fonts, but can be updated in a follow up request if you'd like.

If you have any good ones, then feel free to add them. 😃 

If you can't put names, it would be preferable to leave them as something like "New Font (number)" other than as empty strings so that they can be addressed more easily

**Comment by katieleebrown** [comment]
Thanks for the detailed feedback - I'll update them this afternoon and let you know once the changes have been added!

**Comment by katieleebrown** [comment]
Just pushed up a second commit - the fonts from line 248 onwards have been formatted into arrays and names have been added. Also removed the > that was throwing the error for the bracket font. I've given descriptive names (like double underline) to the fonts that seemed immediately obvious, and then updated the remainder with Font Name X. 

A few things to note: 
- The "T" in new Font 9 isn't rendering correctly
- I didn't upload the heart font from the txt file as I wasn't sure of your preferred placement (hearts before, after, or on both sides of the character).

**Review by waterrmalann — APPROVED** [review]
Proper formatting, the fonts now work perfectly.

New Font 9 doesn't render the T properly because there's an unwanted extra character between J and L, making the length of the array 27 instead of 26. But since this is a minor issue, I'll fix that. Great work!

**Comment by waterrmalann** [comment]
Wonderful job, merged! 

> I didn't upload the heart font from the txt file as I wasn't sure of your preferred placement (hearts before, after, or on both sides of the character).

Woops yeah there was a missing heart before the lowercase alphabets. That's fine, this much fonts are enough :)

**Pull request merged**